plugins/connection/lxc.py

```diff
@@ -66,7 +67,7 @@ def _connect(self):
         super(Connection, self)._connect()
 
         if not HAS_LIBLXC:
-            msg = "lxc bindings for python2 are not installed"
+            msg = "lxc bindings for python%s are not installed" % platform.python_version()
```

Comment:
I would simply change this to
```suggestion
            msg = "lxc bindings for Python are not installed"
```
or alternatively use `ansible.module_utils.basic.missing_required_lib()` which inserts all the necessary information.